liferay-echo/liferay-portal
# LPD-17834 Use the compact modal for the creation of the Display Page folders and  show the necessary information in the Folder Info Panel

>[!IMPORTANT]
>You need the FF LPS-189856

## Motivation

[Link to LPD-17833](https://liferay.atlassian.net/browse/LPD-17833)
[Link to LPD-17834](https://liferay.atlassian.net/browse/LPD-17834)

### Changes

Change the modal for the creation of DPT folders to a small size:
![image](https://github.com/liferay-echo/liferay-portal/assets/93203423/851cdb30-9bb4-43d2-a340-5e0c7d4d717f)

And add a button to manage permissions from the info panel:
![image](https://github.com/liferay-echo/liferay-portal/assets/93203423/f966c8bd-07f4-4022-b1b3-4da77365305e)

## Test scenario 

1. Go to Design > Page Templates > Display Page Templates.
2. Create a folder (check the size of the modal).
3. Create a Display Page Template.
4. Use the Info Panel with both (folder and DPT) and check the button behaviour.

https://github.com/liferay-echo/liferay-portal/assets/93203423/6657c183-3908-4e9e-99e8-1e3c78175e62


>[!NOTE]
>The description input will be move to a different story so it won't be needed to check it in this pr. And this changes not need to add the FF due to the view is not show until it is activated.
